libraries/AP_ServoRelayEvents/AP_ServoRelayEvents.cpp

```diff
@@ -106,10 +106,6 @@ void AP_ServoRelayEvents::update_events(void)
         return;
     }
 
-    if (channel > NUM_SERVO_CHANNELS || channel == 0) {
```

Comment:
Is the protection of channel > NUM_SERVO_CHANNELS not worth it? That part looks like it should still happen to me.